string_reverse.py

```diff
@@ -0,0 +1,3 @@
+# A function to reverse a string.
+def string_reverse(my_string):
+    pass
```

Comment:
It might be worthwhile to have the default implementation of this function be `raise NotImplementedError` instead. Right now this function _does_ pass the test for when `None` is provided as the argument.

Reply:
Good call! Updated to raise the error